tests/src/testUnified.cpp

```diff
@@ -46,7 +49,7 @@ struct ProcessArrayFactoryBackend : DeviceBackendImpl {
 
   void activateAsyncRead() noexcept override {}
 
-  boost::shared_ptr<ProcessArray<std::string>> _pv;
+  std::vector<boost::shared_ptr<ProcessArray<std::string>>> _pv;
```

Comment:
Maybe you need a map<RegisterPath, vector<shared_ptr<ProcessArray>>>. This solves the close() problem and the future problem that multiple variables might be tested in parallel.

Reply:
added